### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by mwsepulveda**
Finished HW2

### CODE REVIEW

**Comment by mwsepulveda** [comment]
Realized that I should change the file paths within the students/mws2170.md file since the paths there were from the mws2170.md file in my repo home directory (I believe this was the instruction from hw0, to place it on its own outside of the students directory). Will points be taken off for this for HW2, or could I fix this later?

File: mws2170.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+|  |  | 
+|:--|:--|
+|Name|Michael Sepulveda|
+|UNI| mws2170|
+| GitHub Profile | [link](https://github.com/mwsepulveda) |
+| GitHub README | [link](https://github.com/mwsepulveda/mwsepulveda/blob/main/README.md) |
+
+
+## Project Proposal
+- [noobJS](./projects/javascript/noobJS.md)
```
Comment (timkpaine) [inline mws2170.md:13]: update link path and re-squash

File: mws2170.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Project Contributions
+- [freeCodeCamp](./projects/javascript/freecodecamp.md)
```
Comment (timkpaine) [inline mws2170.md:16]: update link path and re-squash

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: mws2170.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Open Source Development
```
Comment (timkpaine) [inline mws2170.md:1]: remove file, in `students/` now

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Pull request merged**